crates/runtime/src/http/routes.rs

```diff
@@ -236,65 +236,3 @@ fn cors_layer(cors_config: &CorsConfig) -> CorsLayer {
     cors.allow_methods([Method::GET, Method::POST, Method::PATCH])
         .allow_origin(allowed_origins)
 }
-
```

Comment:
This test is kind of pointless. No good way of programmatically doing it based on `axum::Router`.